modules/text-input/react/lib/TextInput.tsx

```diff
@@ -1,11 +1,11 @@
 import * as React from 'react';
```

Comment:
This looks like a prettier/format only change right?

Reply:
Oh, yes. I'll remove this from the PR

Reply:
A bit tricky as precommit wanted to fix, but I don't want conflicts with other changes